packages/token-vault/src/lib/client.ts

```diff
@@ -88,18 +116,35 @@ export function client(config: ClientConfig) {
         };
       });
     },
+
+    /** ****************************************************
+     * @method store - Register the Token Vault Store
+     * @returns {Promise<Tokens>} - The Token Vault token store object
+     * @example const tokenStore = register.store();
+     */
     store: function () {
       const clientId = config?.forgerock?.clientId || 'WebOAuthClient';
-      const hasTokenEventName = config?.events?.has || 'HAS_TOKENS';
-      const refreshTokenEventName = config?.events?.refresh || 'REFRESH_TOKENS';
-      const removeTokenEventName = config?.events?.remove || 'REMOVE_TOKENS';
+      const hasTokenEventName = config?.events?.has || 'TVP_HAS_TOKENS';
+      const refreshTokenEventName =
+        config?.events?.refresh || 'TVP_REFRESH_TOKENS';
+      const removeTokenEventName =
+        config?.events?.remove || 'TVP_REMOVE_TOKENS';
 
       return {
-        get() {
+        /**
+         * @method get - A noop method that replaces the default get method
+         * @returns {Promise<null>}
+         */
+        get(clientId: string): Promise<Tokens> {
           // Tokens are not retrievable from the iframe
           return Promise.resolve(null as unknown as Tokens);
         },
-        has() {
+
+        /**
+         * @method has - Check if tokens exist in the Token Vault
+         * @returns {Promise<{ hasTokens: boolean }>} - True if tokens exist, false if not
+         */
+        has(): Promise<{ hasTokens: boolean }> {
```

Comment:
You will notice the two additional methods for the main app to directly ask the Proxy a question or request an action. Notice that none return tokens.

This one is just asking the Proxy if it has tokens, and it returns `{ hasTokens: boolean }`.